language/Syntax.md

```diff
@@ -415,13 +415,13 @@ factorial =
     go 1
 ```
 
-The `where` keyword can also be used to introduce local declarations at the end of a value declaration:
+`where` clauses can be added at the end of a function declaration, and allow to bind variables that are visible throughout the function:
```

Comment:
I think this sentence was okay before; what was your reasoning behind this change?